resources/styles/global/typography.less

```diff
@@ -41,6 +41,8 @@
     font-style: normal;
     font-size: @size;
     line-height: @line-height;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
Does it break if we add it, if not, then probably? It seems like the rest of the vendor-prefixes we use also contain the non-prefixed versions (so browsers *just work* once the rule is officially adopted)